hydradx/notebooks/Basilisk/BasicModel.ipynb

```diff
@@ -0,0 +1,278 @@
+      "text/plain": [
+       "<Figure size 1440x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(20, 5))\n",
```

Comment:
Ok, I can clarify that. And the range is actually much larger than it appears, I had just distorted it because the spread was not very even. It is actually similar to what you said. But yeah, that could be a lot clearer.